test/spec/unit/core/targeting_spec.js

```diff
@@ -330,6 +330,73 @@ describe('targeting tests', function () {
       expect(logErrorStub.calledOnce).to.be.true;
     });
 
+    describe('targetingControls.alwaysIncludeDeals', function () {
+      let bid4;
+
+      let targetingWithAdnxsDeal = {
+        hb_deal_rubicon: '1234',
+        hb_deal: '1234',
+        hb_pb: '0.53',
+        hb_adid: '148018fe5e',
+        hb_bidder: 'rubicon',
+        foobar: '300x250',
+        hb_deal_appnexus: '1234',
+        hb_pb_appnexus: '0.53',
+        hb_adid_appnexus: '148018fe5e',
+        hb_bidder_appnexus: 'appnexus',
+      };
+      let targetingWithoutAdnxsDeal = {
```

Comment:
I'd also like to see a scenario where the winning bid isn't a deal.

Reply:
Ok Will do!